modules/linear_boltzmann_solvers/discrete_ordinates_solver/sweep/scheduler/sweep_scheduler.cc

```diff
@@ -246,20 +232,14 @@ SweepScheduler::ScheduleAlgoDOG(SweepChunk& sweep_chunk)
       rbndry->ResetAnglesReadyStatus();
     }
   }
-
-  log.LogEvent(sweep_event_tag_, Logger::EventType::EVENT_END);
 }
 
 void
 SweepScheduler::ScheduleAlgoFIFO(SweepChunk& sweep_chunk)
 {
-  typedef AngleSetStatus Status;
-
-  log.LogEvent(sweep_event_tag_, Logger::EventType::EVENT_BEGIN);
-
-  auto ev_info_i = std::make_shared<Logger::EventInfo>(std::string("Sweep initiated"));
+  CALI_CXX_MARK_SCOPE("SweepScheduler::ScheduleAlgoFIFO");
 
-  log.LogEvent(sweep_event_tag_, Logger::EventType::SINGLE_OCCURRENCE, ev_info_i);
+  typedef AngleSetStatus Status;
```

Comment:
Can you get rid of this? I see some code below already using `AngleSetStatus`

Reply:
Fixed throughout the file. I'll see if I can refactor the reaming pieces of `sweep` this week.